core/lib/dal/src/models/storage_protocol_version.rs

```diff
@@ -63,7 +63,7 @@ impl From<StorageProtocolVersion> for api::ProtocolVersion {
             .as_ref()
             .map(|hash| H256::from_slice(hash));
         api::ProtocolVersion {
-            version_id: storage_protocol_version.id as u16,
+            version_id: storage_protocol_version.minor as u16,
```

Comment:
I wonder if we shouldn't add the 'patch' info to api::ProtocolVersion too.

Reply:
As I've said, I wasn't changing API types for now